pkg/services/authn/authnimpl/sync/org_sync.go

```diff
@@ -29,23 +29,25 @@ func (s *OrgSync) SyncOrgRolesHook(ctx context.Context, id *authn.Identity, _ *a
 		return nil
 	}
 
+	ctxLogger := s.log.FromContext(ctx)
+
 	namespace, userID := id.NamespacedID()
 	if namespace != authn.NamespaceUser || userID <= 0 {
-		s.log.Warn("invalid namespace %q for user ID %q", namespace, userID)
+		ctxLogger.Warn("Failed to sync org role, invalid namespace for identity", "id", id.ID)
```

Comment:
nit: could also append the namespace to the log

```suggestion
		ctxLogger.Warn("Failed to sync org role, invalid namespace for identity", "id", id.ID, "namespace", namespace)
```